tests/prepare_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+cd ..
+make clean
+make DEBUG=1  # compile optionally with PRINTF
```

Comment:
please add `-j` to make this fast :pray:

Reply:
Done in eb1f8041.